src/com/ecommerce/app/service/CartService.java

```diff
@@ -0,0 +1,23 @@
+package com.ecommerce.app.service;
+
+import com.ecommerce.app.App;
+import com.ecommerce.app.model.AddProduct;
+import com.ecommerce.app.model.Cart;
+
+public class CartService {
+
+    private Cart cart = new Cart();
```

Comment:
the add, delete logic should be done in Cart Service. Cart is only a model object used for carrying data